gradle.properties

```diff
@@ -24,4 +24,4 @@ kotlin.stdlib.default.dependency=false
 nodeBinaries.commit=8755ae4c05fd476cd23f2972049111ba436c86d4
 nodeBinaries.version=v20.12.2
 cody.autocomplete.enableFormatting=true
-cody.commit=a7ba3c987888439299edb6c215bb8b3ed4765d1d
+cody.commit=93cb98ba48f935949af0af93c6219b1b7e9076ef
```

Comment:
I merged your other PR to Cody, I guess we need to update commit hash?

Reply:
yes